### ISSUE TITLE
Wrong accessibility in ion-label with Angular expressions

### ISSUE THREAD

**Issue by dpalou** [ionic-team/ionic#9334]
**Ionic version:**  (check one with "x")
[x] **1.x**
[ ] **2.x**

**I'm submitting a ...**  (check one with "x")
[x] bug report
[ ] feature request
[ ] support request => Please do not submit support requests here, use one of these channels: https://forum.ionicframework.com/ or http://ionicworldwide.herokuapp.com/

**Current behavior:**
If I use ion-label with some Angular expressions, the expressions are copied in the aria-label as they are, without being evaluated. Example:
```
<ion-label>{{ 'mm.login.username' | translate }}</ion-label>
```
In the DOM it ends up being:
```
<ion-label class="ng-binding input-label" aria-label="{{ 'mm.login.username' | translate }}" id="_label-3">Username</ion-label>
```
**Expected behavior:**
Ideally, the Angular expressions should be evaluated, so if I write:
```
<ion-label>{{ 'mm.login.username' | translate }}</ion-label>
```
The result should be:
```
<ion-label class="ng-binding input-label" aria-label="Username" id="_label-3">Username</ion-label>
```
**Steps to reproduce:**
http://plnkr.co/edit/7233M24HFjbKD4zOvd6p?p=preview

**Related code:**
The problem is caused by this line:
```
$element.attr('aria-label', $element.text());
```

**Other information:**

**Ionic info:** (run `ionic info` from a terminal/cmd prompt and paste output below):

```
Cordova CLI: 6.1.1
Gulp version:  CLI version 1.2.2
Gulp local:   Local version 3.9.1
Ionic Version: 1.3.0
Ionic CLI Version: 1.7.14
Ionic App Lib Version: 0.7.0
ios-deploy version: 1.8.5 
ios-sim version: 5.0.8 
OS: Mac OS X El Capitan
Node Version: v6.9.1
Xcode version: Xcode 8.1 Build version 8B62 
```
